File: skill_sdk/tests/fixtures/js-project/src/core/FinalizeOutput.js
/**
 * Applies post-processing to the result. Mirrors FinalizeOutput in the Java fixture.
 */
export class FinalizeOutput {
  constructor() {
    throw new Error('Utility class — do not instantiate');
  }

  /**
   * @param {string} data
   * @returns {string}
   */
  static finalize(data) {
    return `[final] ${data} [ok]`;
  }
}

File: skill_sdk/tests/fixtures/java-project/src/main/java/com/fixture/core/FinalizeOutput.java
package com.fixture.core;

/**
 * Applies post-processing to the result. Mirrors FinalizeOutput in main.go.
 */
public final class FinalizeOutput {

    private FinalizeOutput() {
    }

    public static String finalize(String data) {
        return "[final] " + data + " [ok]";
    }
}
